### DISCUSSION TITLE
Auto update task scheduler script

### DISCUSSION THREAD

**Discussion by chipmunk87** [ScoopInstaller/Scoop#5145] [Help and Questions] [answered]
How can I set up an auto update script for Microsoft Task Scheduler?  Is it as simple as running powershell.exe and putting in scoop update * for optional parameters?

**Comment by rashil2000** [comment, marked as answer]
See https://github.com/ScoopInstaller/Scoop/issues/3633

**Comment by chipmunk87** [comment]
Thanks!
